battleship.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Battleship Game
+
+[Service]
+Type=simple
+User=battleship
+Group=battleship
+Restart=on-failure
+Environment=MIX_ENV=prod "PORT=4793"
+Environment=LANG=en_US.UTF-8
+
+WorkingDirectory=/home/battleship/battleship
+ExecStart=/home/battleship/battleship/start.sh
```

Comment:
We'll still need to create the `start.sh` script that's referenced here, but otherwise looks good!